### ISSUE TITLE
Some parameters not selectable in Ableton

### ISSUE THREAD

**Issue by adonaldson** [asb2m10/dexed#274]
Hi there - apologies for the "this doesn't work in application X" sort of post, but I'm not knowledgable enough in VST dev so was hoping someone more experienced could help me flesh out this issue.

In Ableton you can "configure" a VST, which makes the parameters available to control outside of the VST's interface. This also lets you map those controls to MIDI. I'm pretty sure Dexed supports mapping controls to MIDI, but using Ableton in this case allows you to persist the settings.

During the "configure" process, you click on the VST controls and it adds them to the list of parameters. I've noticed that there are a few controls missing. For instance, on operator 2, the following controls can't be "configured":

- OP2 Tune
- OP2 Coarse tune
- OP2 EG Levels (1-4)
- OP2 Output level
- OP2 R Depth
- OP2 Mod sense
- OP2 Mode (fixed/ratio)

I'm running Windows 10, Dexed 1.9.5 with Ableton Live lite.

It's very possible this is user error, or an Ableton bug, however I saw similar problems on my (aging) Mac, which seemed to go away as soon as I used the AU instead of the VST, which made me wonder if it's related to #266.

If anyone thinks this is a problem with Dexed and not my setup, I'm happy to go and build a list of all of the parameters that need "fixed" - just let me know!

Thanks!

**Comment by adonaldson** [comment]
Some extra information on this. I don't have development tools on my Windows 10 machine, so cloned the repo and compiled master on a Linux VM.

With the 0.9.5 VST installed, I was able to add OP2 EG Levels 1-2 as parameters, but any changes I made didn't affect Dexed (audiibly/visually).

With the master build VST installed, I was able to add OP2 EG Levels 1-2 as parameters, and changes did affect Dexed (with real time UI updates).

Is there anywhere I can download a nightly build for Windows to carry on my sleuthing?
